lib/ansible/cli/galaxy.py

```diff
@@ -498,6 +498,8 @@ def add_install_options(self, parser, parents=None):
             install_parser.add_argument('--ignore-signature-status-code', dest='ignore_gpg_errors', type=str, action='append',
                                         help=ignore_gpg_status_help, default=C.GALAXY_IGNORE_INVALID_SIGNATURE_STATUS_CODES,
                                         choices=list(GPG_ERROR_MAP.keys()))
+            install_parser.add_argument('--init-submodules', dest='init_submodules', action='store_true',
+                                        default=False, help='Init collection submodules')
```

Comment:
This should probably have a mention of Git in the option name and explanation that it's only used when installing from Git SCM and doesn't do anything otherwise.